01 - Community/Obsidian Roundup/2023-07-01 n Hiatus Due To Impending Baby.md

```diff
@@ -0,0 +1,28 @@
+---
+link: https://www.eleanorkonik.com/hiatus-announcement/
+author: Eleanor Konik
+published: 2023-07-01T12:30:35
+publish: true
+---
+
+# 2023-07-01: n Hiatus Due To Impending Baby
```

Comment:
The title should be `On` - but if I rename it, this file will get recreated on the next run... So I propose to leave it as it is.